.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+.env
```

Comment:
Be sure to include the `.gitignore` file early on your project so that you don't get some files/directories tracked and then exclude them from the git, because there will be already tracked at this point.